Automattic/jetpack
# Slideshow: Add better email styles for slideshow blocks

#### Changes proposed in this Pull Request:

* Add a custom media query with styles for slideshows that are sent out via email notifications.

**Original**

<img width="714" alt="Screen Shot 2020-10-29 at 12 39 34 PM" src="https://user-images.githubusercontent.com/2124984/97604397-d2cf3e80-19e3-11eb-912e-530a09b3e8f7.png">

**Before**

<img width="619" alt="Screen Shot 2020-10-29 at 12 28 20 PM" src="https://user-images.githubusercontent.com/2124984/97602915-46704c00-19e2-11eb-843d-12937967e39d.png">

**After**

<img width="612" alt="Screen Shot 2020-10-29 at 12 27 28 PM" src="https://user-images.githubusercontent.com/2124984/97602931-4b350000-19e2-11eb-90f7-bbabd94016cf.png">

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Apply this patch to your WP.com sandbox and sandbox a test site.
* Subscribe to/follow the test site and make sure email notifications are turned on under Reader -> Manage Subscriptions.
* Create a post with a Jetpack Slideshow on the test site you've subscribed to.
* You should receive an email notification with the slideshow contents shown as a gallery (like above).

#### Proposed changelog entry for your changes:

* Add email styles for slideshows
